posthog/tasks/usage_report.py

```diff
@@ -516,6 +506,19 @@ def has_non_zero_usage(report: FullUsageReport) -> bool:
     )
 
 
+def convert_team_usage_rows_to_dict(rows: List[Union[dict, Tuple[int, int]]]) -> Dict[int, int]:
+    team_id_map = {}
+    for row in rows:
+        if "team_id" in row:
+            # Some queries return a dict with team_id and total
+            team_id_map[row["team_id"]] = row["total"]
```

Comment:
I wonder if we should standardize how the data is returned so things don't get dropped here in the future

Reply:
Yeah we definitely should. I think the only cases where this happens currently is for Postgres calls. I might actually make that change now